latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_common-1.0.0
+
+title: All PIT common metadata
+description: |
+  This schema is the union of all Roman PIT common metadata, including
+  those found in the referenced pit_basic schema.
+type: object
+allOf:
+  - $ref: asdf://stsci.edu/datamodels/roman/schemas/PIT/pit_basic-1.0.0
+  - type: object
+    properties:
+      file_date:
+        title: File creation date
+        description: |
+          File creation date-time in ISO 8601 format with UTC time
+          scale.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      start_time:
+        title: Start time
+        description: |
+          Start time of data acquisition in ISO 8601 format.
+          tag: tag:stsci.edu:asdf/time/time-1.*
```

```suggestion
        tag: tag:stsci.edu:asdf/time/time-1.*
```
What's the use in having `start_time` and `start_time_mjd`?

---

Double-checking with CAOM folks now but we can probably drop start_time_mjd, yes.